goatboard-api/src/main/kotlin/com/github/netbenix/routing/PostPostsRouting.kt

```diff
@@ -0,0 +1,42 @@
+package com.github.netbenix.routing
+
+import com.github.netbenix.datatypes.json.PostRequestBody
+import com.github.netbenix.managers.DBManager
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import io.ktor.util.logging.*
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import java.util.*
+import kotlin.NoSuchElementException
+
+fun Route.postPostsRouting() {
+    post("/{boardId}/post"){
+        val logger = KtorSimpleLogger("com.github.netbenix.goatboard-api.API")
+
+        try {
+            val reqBody = Json.decodeFromString<PostRequestBody>(call.receiveText())
+
+            DBManager.dbInstance.getBoard(UUID.fromString(call.parameters["boardId"]))
+            DBManager.dbInstance.createPost(UUID.fromString(call.parameters["boardId"]), reqBody.title, reqBody.message)
+            call.respond(HttpStatusCode.Created)
+        } catch (ex: NoSuchElementException) {
+            logger.info("Board '${call.parameters["boardID"]}' was requested. But not found!")
```

Comment:
Please be cautious of the casing to keep consistency
```suggestion
            logger.info("Board '${call.parameters["boardId"]}' was requested. But not found!")
```